### DISCUSSION TITLE
Singularity joins the Linux Foundation and is rebranded to be Apptainer

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#110] [General]
Wasn't aware of that:
https://apptainer.org/news/community-announcement-20211130/

What does it mean for us? Should we stick with Sylabs' Singularity or rather Apptainer, assuming they will diverge in functionality with time?

**Comment by stebo85** [comment]
Yes, I am following this closely. We will stick with Singularity for now, because it all works and see how this situation unfolds. Changing to apptainer would be trivial, but not worth the time right now.
